### PR TITLE
Fair QA fixes

### PR DESCRIPTION
**Pull request by kierangillen**
- Fixes inconsistent padding between items in show and fair views
- Removes fair save button
- Adds params to fair link
- Fixes Exhibitor name overflows and pushes follow button off the screen ( https://artsyproduct.atlassian.net/browse/LD-206 )

## Removes save button

screen shot 2019-02-15 at 1 38 57 pm

## Fixes exhibitor overflow

screen shot 2019-02-15 at 2 53 55 pm

### CODE REVIEW

File: src/lib/Scenes/Fair/Components/FairHeader/index.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Box, Flex, Sans, space, Spacer } from "@artsy/palette"
 import { FairHeader_fair } from "__generated__/FairHeader_fair.graphql"
 import { FairHeaderMutation } from "__generated__/FairHeaderMutation.graphql"
-import InvertedButton from "lib/Components/Buttons/InvertedButton"
+// import InvertedButton from "lib/Components/Buttons/InvertedButton"
```
Comment (ashfurrow) [inline src/lib/Scenes/Fair/Components/FairHeader/index.tsx]: We should remove this if it's no longer needed.

File: src/lib/Scenes/Fair/Components/FairHeader/index.tsx [hunk 2]
```diff
@@ -162,7 +162,8 @@ export class FairHeader extends React.Component<Props, State> {
     } = this.props
     const { width: screenWidth } = Dimensions.get("window")
     const imageHeight = 567
-    const { isSavedFairStateUpdating } = this.state
+    // const { isSavedFairStateUpdating } = this.state
+    // FIXME: When saving fairs is defined we can use commented code
```
Comment (ashfurrow) [inline src/lib/Scenes/Fair/Components/FairHeader/index.tsx]: I'd prefer if we removed the `isSavedFairStateUpdating` state value and `handleSaveFair`. If it's not getting called, it should be removed – but we can still access it in git history.

We should add a reference to this pull request in the ticket to re-add the fair saving, though, to make it easy to add back in 👍

File: src/lib/Scenes/Home/Components/ForYou/Components/FairsRail.tsx [hunk 3]
```diff
@@ -58,7 +58,7 @@ export class FairsRail extends Component<Props, null> {
       }
 
       const selectionHandler = () => {
-        Switchboard.presentNavigationViewController(this, fair.profile.href)
+        Switchboard.presentNavigationViewController(this, `${fair.profile.href}?entity=fair`)
```
Comment (ashfurrow) [inline src/lib/Scenes/Home/Components/ForYou/Components/FairsRail.tsx]: Exxxxxxcellent

File: src/lib/Scenes/Fair/Components/FairHeader/index.tsx (hunk 2 above)
Comment (kierangillen) [reply src/lib/Scenes/Fair/Components/FairHeader/index.tsx]: ok sounds good

**Pull request merged**